stdlib/public/core/Slice.swift

```diff
@@ -135,6 +135,9 @@ public struct Slice<Base: Collection> {
   public var base: Base {
     return _base
   }
+
+  /// Test cost of making Slice artificially address-only
```

Comment:
It would be better to add an underscored attribute for this.

Reply:
Sure, I'm just trying an experiment to see if it's even viable. Adding an `Any` field also approximates making the value witness operations opaque.

Reply:
Well, a worst case test would be to just remove the @_fixed_layout attribute wouldn’t it?